test/unit/ObjectMakrTests/make.js

```diff
@@ -0,0 +1,38 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("copies a string", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            // Act
+            var madeObject = objectMaker.make(mocks.mockClassName, mocks.mockObjectProperties());
+
+            // Assert
+            expect(typeof madeObject.name).to.equal("string");
+        });
+
+        it("implements an onMake function", function () {
```

Comment:
The other two test names are good, but this still is implementation-specific. How about changing to "copies a function"?